apps/web/src/components/PlanningWrapper.tsx

```diff
@@ -2,13 +2,16 @@ import React, { useState } from 'react';
 import { Stepper, Button, PlanningContent } from '@components';
 import { Formik, Form } from 'formik';
 import { PlanningSteps } from '../common/constants';
+// import { usePlanningSession } from '../services/usePlanningSession';
 
 interface PlanningWrapperFormValues {
   firstName: string;
   lastName: string;
 }
 
 export const PlanningWrapper = () => {
+  // const { planningSession } = usePlanningSession();
```

Comment:
Can be uncommented. This should be called each time we land here.

Reply:
Thanks so much Karthikk! ESLint won't allow me to commit because if I uncomment it will say "variable assigned but never used". Should I do something useless with it as a workaround or comment for now? Thanks in advance!